### DISCUSSION TITLE
baseline image of the Score-base CAM

### DISCUSSION THREAD

**Discussion by ROOKLO** [frgfm/torch-cam#147] [Q&A] [answered]
the papers of score-based cam usually use a baseline image to calculate channel-wise increase of confidence(CIC). But what is the baseline image used in the torch-cam?

**Comment by frgfm** [comment, marked as answer]
Hi @ROOKLO :wave: 

Thanks for the question! It's very relevant: it's my own interpretation but the author doesn't mention any specific image as reference, so I took the liberty of using the same as the input image. This seems to be aligned with the reasoning of the author since he published his implementation after I added it to TorchCAM, and the input image was also used :+1: 

Happy to discuss that further if you have suggestions about this!

Cheers :v:
